plugins/woocommerce-blocks/assets/js/blocks/order-confirmation/create-account/style.scss

```diff
@@ -74,6 +74,10 @@
 					white-space: nowrap;
 				}
 			}
+
+			p + p {
+				margin-top: $gap-small;
```

Comment:
How so? If you add more paragraphs in the future they will be styled correctly with this rule in place ^ Not everything needs a class. 

Does not matter anyway; design came back in the issue and want a single paragraph so Ill remove this and the 2 paragraphs.

Reply:
https://github.com/woocommerce/woocommerce/pull/53558/commits/03485de954472bbec1feeb887f1bdcce4f4f0799

Reply:
> How so? If you add more paragraphs in the future they will be styled correctly with this rule in place ^ Not everything needs a class.

@mikejolley As a general approach in CSS it is best to avoid element targets. I can give so many reasons for this and in this one case it might not matter so much but even scoped to a class name like it is here it is not great when you adjust structure in future. 

> If you add more paragraphs in the future they will be styled correctly with this rule in place ^ Not everything needs a class.

You've covered exactly one case that will work but if we used other semantic tags it would fail. I understand it's a nit pick but I'd just argue there are very few cases we should be targeting tag names.

The more complex the styles become in one area the more these tag based selectors will frustrate the next person, they're too vague and easy to affect other areas of the structure, they're styling structure rather than semantics too and that's really the biggest downfall of the approach.

There are other ways this could be adjusted too like we could refactor the margins a little more so we don't have to rely on `p + p` I really think we shouldn't be styling that way. It's too rigid to the structure.